# HG changeset patch
# User Andreas Weber <andy.weber.aw@gmail.com>
# Date 1424200409 -3600
#      Tue Feb 17 20:13:29 2015 +0100
# Node ID 9861618b4c5fe96ed04d7a7a6c5b69004c6c8408
# Parent  554aaaf99644febc1d02ac6b3a86354207591d0e
libgui/languages/de_DE.ts: Update german translation

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -72,17 +72,17 @@
     <message>
         <location line="+4"/>
         <source>Paste</source>
         <translation>Einfügen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Select All</source>
-        <translation type="unfinished">Alles auswählen</translation>
+        <translation>Alles auswählen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Clear All</source>
         <translation>Alles löschen</translation>
     </message>
 </context>
 <context>
@@ -98,115 +98,115 @@
     <message>
         <location filename="../graphics/Figure.cc" line="+254"/>
         <source>Figure ToolBar</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+20"/>
         <source>&amp;File</source>
-        <translation type="unfinished">&amp;Datei</translation>
+        <translation>&amp;Datei</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;New Figure</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Neue Abbildung</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>&amp;Open...</source>
-        <translation type="unfinished"></translation>
+        <translation>Ö&amp;ffnen...</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Save</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Speichern</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Save &amp;As</source>
-        <translation type="unfinished"></translation>
+        <translation>Speichern &amp;unter</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Close Figure</source>
-        <translation type="unfinished"></translation>
+        <translation>Abbildung s&amp;chließen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Edit</source>
-        <translation type="unfinished">&amp;Bearbeiten</translation>
+        <translation>&amp;Bearbeiten</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Cop&amp;y</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Kopieren</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Cu&amp;t</source>
-        <translation type="unfinished">&amp;Ausschneiden</translation>
+        <translation>&amp;Ausschneiden</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Paste</source>
-        <translation type="unfinished"></translation>
+        <translation>E&amp;infügen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Help</source>
-        <translation type="unfinished">&amp;Hilfe</translation>
+        <translation>&amp;Hilfe</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;About QtHandles</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Über QtHandles</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>About &amp;Qt</source>
-        <translation type="unfinished"></translation>
+        <translation>Über &amp;Qt</translation>
     </message>
     <message>
         <location line="+403"/>
         <source>About QtHandles</source>
-        <translation type="unfinished"></translation>
+        <translation>Über QtHandles</translation>
     </message>
     <message>
         <location line="+41"/>
         <source>Save Figure As</source>
-        <translation type="unfinished"></translation>
+        <translation>Abbildung speichern unter</translation>
     </message>
 </context>
 <context>
     <name>QtHandles::MouseModeActionGroup</name>
     <message>
         <location filename="../graphics/MouseModeActionGroup.cc" line="+40"/>
         <source>Rotate</source>
-        <translation type="unfinished"></translation>
+        <translation>Rotieren</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Zoom</source>
-        <translation type="unfinished"></translation>
+        <translation>Zoomen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Pan</source>
-        <translation type="unfinished"></translation>
+        <translation>Schwenken</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Insert Text</source>
-        <translation type="unfinished"></translation>
+        <translation>Text einfügen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Select</source>
-        <translation type="unfinished"></translation>
+        <translation>Auswählen</translation>
     </message>
 </context>
 <context>
     <name>documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+34"/>
         <source>Documentation</source>
         <translation>Dokumentation</translation>
@@ -348,22 +348,22 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+2"/>
         <source>&amp;Uncomment</source>
         <translation>Kommentar &amp;entfernen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Indent</source>
-        <translation type="unfinished"></translation>
+        <translation>E&amp;inzug erhöhen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Unindent</source>
-        <translation type="unfinished"></translation>
+        <translation>Einzug verringern</translation>
     </message>
     <message>
         <location line="-55"/>
         <source>&amp;Find and Replace...</source>
         <translation>&amp;Suchen und Ersetzen...</translation>
     </message>
     <message>
         <location line="+164"/>
@@ -379,22 +379,22 @@ ist im Editor bereits geöffnet</translation>
         <location line="-1133"/>
         <source>Could not start custom file editor
 %1</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+148"/>
         <source>Create</source>
-        <translation type="unfinished"></translation>
+        <translation>Erzeugen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Cancel</source>
-        <translation type="unfinished">Abbrechen</translation>
+        <translation>Abbrechen</translation>
     </message>
     <message>
         <location line="+848"/>
         <source>&amp;Recent Editor Files</source>
         <translation>&amp;Zuletzt bearbeitete Dateien</translation>
     </message>
     <message>
         <location line="+8"/>
@@ -414,107 +414,107 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+3"/>
         <source>Close Other Files</source>
         <translation>Andere Dokumente schließen</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Select All</source>
-        <translation type="unfinished">Alles auswählen</translation>
+        <translation>Alles auswählen</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>&amp;Commands</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehle</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Delete Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeile löschen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Copy Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeile kopieren</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Cut Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeile ausschneiden</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Delete to Start of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>Bis zum Anfang des Wortes löschen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Word</source>
-        <translation type="unfinished"></translation>
+        <translation>Bis zum Ende des Wortes löschen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to Start of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Bis zum Anfang der Zeile löschen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete to End of Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Bis zum Ende der Zeile löschen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Duplicate Selection/Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeile oder Auswahl duplizieren</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Transpose Line</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilen tauschen</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>&amp;Show Completion List</source>
-        <translation type="unfinished"></translation>
+        <translation>Vervoll&amp;ständigungsliste anzeigen</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Format</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Format</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>&amp;Uppercase Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Lowercase Selection</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+19"/>
         <source>Convert Line Endings to &amp;Windows (CRLF)</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden für &amp;Windows (CRLF) umwandeln</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden für &amp;Unix (LF) umwandeln</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Convert Line Endings to &amp;Mac (CR)</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden für &amp;Mac (CR) umwandeln</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Navi&amp;gation</source>
-        <translation type="unfinished"></translation>
+        <translation>Navi&amp;gation</translation>
     </message>
     <message>
         <location line="+8"/>
         <source>Move to Matching Brace</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
@@ -529,42 +529,42 @@ ist im Editor bereits geöffnet</translation>
     <message>
         <location line="+3"/>
         <source>&amp;Styles Preferences...</source>
         <translation>&amp;Stil-Einstellungen...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>&amp;View</source>
-        <translation type="unfinished"></translation>
+        <translation>Ansicht</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>&amp;Editor</source>
-        <translation type="unfinished"></translation>
+        <translation>&amp;Editor</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Show &amp;Line Numbers</source>
-        <translation type="unfinished"></translation>
+        <translation>Zei&amp;lennummern anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;White Spaces</source>
-        <translation type="unfinished"></translation>
+        <translation>Leerzeichen anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Line &amp;Endings</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show &amp;Indentation Guides</source>
-        <translation type="unfinished"></translation>
+        <translation>Einrückungshinweise anzeigen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Show Long Line &amp;Marker</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
@@ -676,17 +676,17 @@ konnte nicht zum Schreiben geöffnet werden:
     <message>
         <location line="+145"/>
         <source>%1 is a built-in function</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Can not find function %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Kann Funktion %1 nicht finden</translation>
     </message>
     <message>
         <location line="+769"/>
         <source>Goto line</source>
         <translation>Gehe zu Zeile</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -701,32 +701,36 @@ konnte nicht zum Schreiben geöffnet werden:
     </message>
     <message>
         <location line="+4"/>
         <source>The file
 
   %1
 
 is about to be closed but has been modified.  %2</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Die Datei
+
+  %1
+
+soll geschlossen werden aber wurde modifiziert.  %2</translation>
     </message>
     <message>
         <location line="+306"/>
         <source>Line Endings:</source>
-        <translation type="unfinished"></translation>
+        <translation>Zeilenenden:</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>Octave Files (*.m)</source>
-        <translation type="unfinished"></translation>
+        <translation>Octave Skripte (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>All Files (*)</source>
-        <translation type="unfinished"></translation>
+        <translation>Alle Dateien (*)</translation>
     </message>
     <message>
         <location line="-340"/>
         <source>Do you want to cancel closing, save or discard the changes?</source>
         <translation>Änderungen speichern, verwerfen oder Schließen abbrechen?</translation>
     </message>
     <message>
         <location line="-968"/>
@@ -911,22 +915,22 @@ Do you want to choose another name?</sou
     <message>
         <location line="+6"/>
         <source>Set Current Directory</source>
         <translation>Aktuelles Verzeichnis setzen</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Rename...</source>
-        <translation type="unfinished"></translation>
+        <translation>Umbenennen...</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Delete...</source>
-        <translation type="unfinished"></translation>
+        <translation>Löschen...</translation>
     </message>
     <message>
         <location line="+115"/>
         <source>Rename file/directory</source>
         <translation>Datei/Verzeichnis umbenennen</translation>
     </message>
     <message>
         <location line="+1"/>
@@ -1059,17 +1063,17 @@ a:link { text-decoration: underline; col
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
 </context>
 <context>
     <name>find_dialog</name>
     <message>
         <location filename="../src/m-editor/find-dialog.cc" line="+74"/>
         <source>Find and Replace</source>
-        <translation type="unfinished"></translation>
+        <translation>Suchen und Ersetzen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Find &amp;what:</source>
         <translation>&amp;Suche:</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1343,17 +1347,17 @@ a:link { text-decoration: underline; col
     <message>
         <location line="+8"/>
         <source>Enter text to filter the command history</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Filter</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Command History</source>
         <translation>Befehlshistorie</translation>
     </message>
     <message>
         <location line="+88"/>
@@ -1494,37 +1498,37 @@ instead?</source>
     <message>
         <location line="+192"/>
         <source>Octave Community News</source>
         <translation>Neuigkeiten aus der Octave Community</translation>
     </message>
     <message>
         <location line="+173"/>
         <source>Are you sure you want to exit Octave?</source>
-        <translation type="unfinished"></translation>
+        <translation>Octave wirklich beenden?</translation>
     </message>
     <message>
         <location line="+840"/>
         <source>Open an existing file in editor</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+57"/>
         <source>New Function...</source>
-        <translation type="unfinished"></translation>
+        <translation>Neue Funktion...</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>New Figure</source>
-        <translation type="unfinished"></translation>
+        <translation>Neue Abbildung</translation>
     </message>
     <message>
         <location line="+36"/>
         <source>Select All</source>
-        <translation type="unfinished">Alles auswählen</translation>
+        <translation>Alles auswählen</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Clear Clipboard</source>
         <translation>Zwischenablage leeren</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1575,32 +1579,32 @@ instead?</source>
         <location line="-84"/>
         <source>Step In</source>
         <translation>Hineinspringen</translation>
     </message>
     <message>
         <location line="-1457"/>
         <location line="+455"/>
         <source>Octave</source>
-        <translation type="unfinished"></translation>
+        <translation>Octave</translation>
     </message>
     <message>
         <location line="+854"/>
         <source>Load Workspace...</source>
         <translation>Lade Arbeitsumgebung...</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Save Workspace As...</source>
         <translation>Arbeitsumgebung speichern als...</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>New Script</source>
-        <translation type="unfinished"></translation>
+        <translation>Neues Skript</translation>
     </message>
     <message>
         <location line="+111"/>
         <source>Step Out</source>
         <translation>Herausspringen</translation>
     </message>
     <message>
         <location line="+94"/>
@@ -1671,17 +1675,17 @@ instead?</source>
         <location line="-1129"/>
         <location line="+1132"/>
         <source>Browse directories</source>
         <translation>Verzeichnis suchen</translation>
     </message>
     <message>
         <location line="-175"/>
         <source>Quit Debug Mode</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Debug Modus beenden</translation>
     </message>
     <message>
         <location line="+38"/>
         <source>&amp;Window</source>
         <translation>&amp;Fenster</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1843,17 +1847,17 @@ oder aktiviere die Internetverbindung in
 </context>
 <context>
     <name>octave_qt_link</name>
     <message>
         <location filename="../src/octave-qt-link.cc" line="+111"/>
         <location line="+6"/>
         <location line="+7"/>
         <source>Create</source>
-        <translation type="unfinished"></translation>
+        <translation>Erzeugen</translation>
     </message>
     <message>
         <location line="-10"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
         <translation>Die Datei
 %1
@@ -1862,17 +1866,17 @@ existiert nicht. Soll sie erzeugt werden
     <message>
         <location line="+3"/>
         <source>Octave Editor</source>
         <translation>Octave Editor</translation>
     </message>
     <message>
         <location line="+211"/>
         <source>The file %1 does not exist in the load path.  To run or debug the function you are editing, you must either change to the directory %2 or add that directory to the load path.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Die Datei %1 existiert nicht im Suchpfad.  Um </translation>
     </message>
     <message>
         <location line="+1"/>
         <source>The file %1 is shadowed by a file with the same name in the load path. To run or debug the function you are editing, change to the directory %2.</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
@@ -1896,17 +1900,17 @@ existiert nicht. Soll sie erzeugt werden
         <translation>Abbrechen</translation>
     </message>
 </context>
 <context>
     <name>octave_txt_lexer</name>
     <message>
         <location filename="../src/m-editor/octave-txt-lexer.cc" line="+39"/>
         <source>Default</source>
-        <translation type="unfinished"></translation>
+        <translation>Standard</translation>
     </message>
 </context>
 <context>
     <name>resource_manager</name>
     <message>
         <location filename="../src/resource-manager.cc" line="+235"/>
         <source>The settings file
 %1
@@ -1994,32 +1998,32 @@ Die Octave Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="+9"/>
         <source>Default eol mode</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+14"/>
         <source>Windows (CRLF)</source>
-        <translation type="unfinished"></translation>
+        <translation>Windows (CRLF)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Mac (CR)</source>
-        <translation type="unfinished"></translation>
+        <translation>Mac (CR)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Unix (LF)</source>
-        <translation type="unfinished"></translation>
+        <translation>Unix (LF)</translation>
     </message>
     <message>
         <location line="+26"/>
         <source>Show horizontal scroll bar</source>
-        <translation type="unfinished"></translation>
+        <translation>Horizontale Bildlaufleiste anzeigen</translation>
     </message>
     <message>
         <location line="+24"/>
         <source>Indent width</source>
         <translation>Einrücken um</translation>
     </message>
     <message>
         <location line="+23"/>
@@ -2049,17 +2053,17 @@ Die Octave Benutzeroberfläche muss jetzt geschlossen werden.</translation>
     <message>
         <location line="+49"/>
         <source>Indentation uses tabs</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+26"/>
         <source>Auto completion</source>
-        <translation type="unfinished"></translation>
+        <translation>Autovervollständigung</translation>
     </message>
     <message>
         <location line="+30"/>
         <source>Match keywords</source>
         <translation>Schlüsselwörter berücksichtigen</translation>
     </message>
     <message>
         <location line="+13"/>
@@ -3248,17 +3252,17 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+4"/>
         <source>Global search</source>
         <translation>Globale Suche</translation>
     </message>
     <message>
         <location line="+18"/>
         <source>Error</source>
-        <translation type="unfinished"></translation>
+        <translation>Fehler</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The info file&lt;p&gt;%1&lt;p&gt;or compressed versions do not exist</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
@@ -3289,17 +3293,18 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+1"/>
         <source>Value</source>
         <translation>Wert</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Attribute</source>
-        <translation type="unfinished"></translation>
+        <translatorcomment>alternativ &quot;Eigenschaft&quot;</translatorcomment>
+        <translation>Attribut</translation>
     </message>
     <message>
         <location line="+109"/>
         <source>Right click to copy, rename, or display</source>
         <translation>Rechtsklick zum Kopieren, Umbenennen oder Anzeigen</translation>
     </message>
     <message>
         <location line="+37"/>
@@ -3318,32 +3323,33 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+1"/>
         <source>View the variables in the active workspace.</source>
         <translation>Variablen einsehen, die sich in der aktiven Arbeitsumgebung befinden.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Enter text to filter the workspace</source>
-        <translation type="unfinished"></translation>
+        <translation>Text eingeben um die Arbeitsumgebung zu filtern</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Filter</source>
-        <translation type="unfinished"></translation>
+        <translation>Filter
+</translation>
     </message>
     <message>
         <location line="+147"/>
         <source>Copy name</source>
-        <translation type="unfinished"></translation>
+        <translation>Namen kopieren</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Copy value</source>
-        <translation type="unfinished"></translation>
+        <translation>Wert kopieren</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>Rename</source>
         <translation>Umbenennen</translation>
     </message>
     <message>
         <location line="+9"/>
@@ -3353,12 +3359,12 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+132"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Einsehen der Variablen der aktiven Arbeitsumgebung.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for variable attributes:</source>
-        <translation type="unfinished"></translation>
+        <translation>Farben für Variablen Eigenschaften:</translation>
     </message>
 </context>
 </TS>
